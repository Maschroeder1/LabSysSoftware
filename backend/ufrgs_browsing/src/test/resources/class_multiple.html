    


<html>
<head><title>UFRGS - Horários e Turmas</title>
    <link href="/Design/formulario.css" rel="stylesheet" type="text/css" />
    <link href="/Design/simples.css" rel="stylesheet" type="text/css" />
    <link href="/Design/texto.css" rel="stylesheet" type="text/css" />
    <link href="/Design/tabela.css" rel="stylesheet" type="text/css" />
</head>

<body>


<!-- Aqui inicio o conteudo desta página -->


    <fieldset class="moldura fieldset-1">
        <legend class="legend-2">&nbsp;&nbsp;Horários e Turmas - 2022/2&nbsp;&nbsp;</legend>
        
        <table class="modelo1" style="width:95%">
            <tr valign="middle" >
                <td class="th1" align="center" width="35%">Atividade de Ensino</td>
                <td class="th1" align="center">Carga<br>Horá-<br>ria</td>
                <td class="th1" align="center">Tur-<br>ma</td>
                <td class="th1" align="center">Vagas<br>Ofere-<br>cidas</td>
                <td class="th1" align="center" width="27%">Horário-Período-Local</td>
                <td class="th1" align="center">Professor(es)</td>
            </tr>

            <tr class="modelo1odd" valign="top"  style="font-size:12px ">
                <td align="left">&nbsp;ALGORÍTMOS E PROGRAMAÇÃO - CIC&nbsp;</td>
                <td align="center">&nbsp;6&nbsp;</td>
                <td align="center">&nbsp;A &nbsp;</td>
                <td align="center">&nbsp;1&nbsp;</td>
                <td align="left"><li>Segunda 13:30 - 15:10 (2)<br><a class='clicavel' href='http://mapa.ufrgs.br/index.php?verb=pan&building=44' target='_blank'>SALA DE AULA 114 - PRÉDIO 43425 4</a></li><li>Quarta 13:30 - 15:10 (2)<br><a class='clicavel' href='http://mapa.ufrgs.br/index.php?verb=pan&building=44' target='_blank'>SALA DE AULA 114 - PRÉDIO 43425 4</a></li><li>Quinta 13:30 - 15:10 (2)<br> </li><li><b></b></li></td>
                <td align="left"><li>MARA ABEL</li></td>
            </tr>
            <tr class="modelo1even" valign="top"  style="font-size:12px ">
                <td align="left">&nbsp;ALGORÍTMOS E PROGRAMAÇÃO - CIC&nbsp;</td>
                <td align="center">&nbsp;6&nbsp;</td>
                <td align="center">&nbsp;B &nbsp;</td>
                <td align="center">&nbsp;0&nbsp;</td>
                <td align="left"><li>Segunda 13:30 - 15:10 (2)<br><a class='clicavel' href='http://mapa.ufrgs.br/index.php?verb=pan&building=44' target='_blank'>SALA DE AULA 114 - PRÉDIO 43425 4</a></li><li>Quarta 13:30 - 15:10 (2)<br><a class='clicavel' href='http://mapa.ufrgs.br/index.php?verb=pan&building=44' target='_blank'>SALA DE AULA 114 - PRÉDIO 43425 4</a></li><li>Quinta 15:30 - 17:10 (2)<br><a class='clicavel' href='http://mapa.ufrgs.br/index.php?verb=pan&building=1885' target='_blank'>Laboratório de Ensino 104 - 43413 4</a></li><li><b></b></li></td>
                <td align="left"><li>MARA ABEL</li></td>
            </tr>
            <tr class="modelo1odd" valign="top"  style="font-size:12px ">
                <td align="left">&nbsp;ALGORÍTMOS E PROGRAMAÇÃO - CIC&nbsp;</td>
                <td align="center">&nbsp;6&nbsp;</td>
                <td align="center">&nbsp;D &nbsp;</td>
                <td align="center">&nbsp;4&nbsp;</td>
                <td align="left"><li>Segunda 15:30 - 17:10 (2)<br><a class='clicavel' href='http://mapa.ufrgs.br/index.php?verb=pan&building=44' target='_blank'>SALA DE AULA 114 - PRÉDIO 43425 4</a></li><li>Quarta 15:30 - 17:10 (2)<br><a class='clicavel' href='http://mapa.ufrgs.br/index.php?verb=pan&building=44' target='_blank'>SALA DE AULA 114 - PRÉDIO 43425 4</a></li><li>Quinta 15:30 - 17:10 (2)<br><a class='clicavel' href='http://mapa.ufrgs.br/index.php?verb=pan&building=1885' target='_blank'>Laboratório de Ensino 102 - 43413 4</a></li><li><b></b></li></td>
                <td align="left"><li>LUCAS MELLO SCHNORR</li><li>MARCELO WALTER</li></td>
            </tr>
            <tr class="modelo1even" valign="top"  style="font-size:12px ">
                <td align="left">&nbsp;ALGORÍTMOS E PROGRAMAÇÃO - CIC&nbsp;</td>
                <td align="center">&nbsp;6&nbsp;</td>
                <td align="center">&nbsp;E &nbsp;</td>
                <td align="center">&nbsp;0&nbsp;</td>
                <td align="left"><li>Terça 13:30 - 15:10 (2)<br><a class='clicavel' href='http://mapa.ufrgs.br/index.php?verb=pan&building=44' target='_blank'>SALA DE AULA 114 - PRÉDIO 43425 4</a></li><li>Quinta 13:30 - 15:10 (2)<br><a class='clicavel' href='http://mapa.ufrgs.br/index.php?verb=pan&building=44' target='_blank'>SALA DE AULA 114 - PRÉDIO 43425 4</a></li><li>Sexta 13:30 - 15:10 (2)<br><a class='clicavel' href='http://mapa.ufrgs.br/index.php?verb=pan&building=1885' target='_blank'>Laboratório de Ensino 101 - 43413 4</a></li><li><b></b></li></td>
                <td align="left"><li>Joel Luis Carbonera</li><li>THIAGO LOPES TRUGILLO DA SILVEIRA</li></td>
            </tr>
            <tr class="modelo1odd" valign="top"  style="font-size:12px ">
                <td align="left">&nbsp;ALGORÍTMOS E PROGRAMAÇÃO - CIC&nbsp;</td>
                <td align="center">&nbsp;6&nbsp;</td>
                <td align="center">&nbsp;F &nbsp;</td>
                <td align="center">&nbsp;11&nbsp;</td>
                <td align="left"><li>Terça 13:30 - 15:10 (2)<br><a class='clicavel' href='http://mapa.ufrgs.br/index.php?verb=pan&building=44' target='_blank'>SALA DE AULA 114 - PRÉDIO 43425 4</a></li><li>Quinta 13:30 - 15:10 (2)<br><a class='clicavel' href='http://mapa.ufrgs.br/index.php?verb=pan&building=44' target='_blank'>SALA DE AULA 114 - PRÉDIO 43425 4</a></li><li>Sexta 15:30 - 17:10 (2)<br><a class='clicavel' href='http://mapa.ufrgs.br/index.php?verb=pan&building=1885' target='_blank'>Laboratório de Ensino 101 - 43413 4</a></li><li><b></b></li></td>
                <td align="left"><li>Joel Luis Carbonera</li><li>THIAGO LOPES TRUGILLO DA SILVEIRA</li></td>
            </tr>
            <tr class="modelo1even" valign="top"  style="font-size:12px ">
                <td align="left">&nbsp;ALGORÍTMOS E PROGRAMAÇÃO - CIC&nbsp;</td>
                <td align="center">&nbsp;6&nbsp;</td>
                <td align="center">&nbsp;G &nbsp;</td>
                <td align="center">&nbsp;14&nbsp;</td>
                <td align="left"><li>Terça 15:30 - 17:10 (2)<br><a class='clicavel' href='http://mapa.ufrgs.br/index.php?verb=pan&building=44' target='_blank'>SALA DE AULA 114 - PRÉDIO 43425 4</a></li><li>Quinta 15:30 - 17:10 (2)<br><a class='clicavel' href='http://mapa.ufrgs.br/index.php?verb=pan&building=44' target='_blank'>SALA DE AULA 114 - PRÉDIO 43425 4</a></li><li>Sexta 13:30 - 15:10 (2)<br><a class='clicavel' href='http://mapa.ufrgs.br/index.php?verb=pan&building=1885' target='_blank'>Laboratório de Ensino 102 - 43413 4</a></li><li><b></b></li></td>
                <td align="left"><li>Renan de Queiroz Maffei</li></td>
            </tr>
            <tr class="modelo1odd" valign="top"  style="font-size:12px ">
                <td align="left">&nbsp;ALGORÍTMOS E PROGRAMAÇÃO - CIC&nbsp;</td>
                <td align="center">&nbsp;6&nbsp;</td>
                <td align="center">&nbsp;H &nbsp;</td>
                <td align="center">&nbsp;10&nbsp;</td>
                <td align="left"><li>Terça 15:30 - 17:10 (2)<br><a class='clicavel' href='http://mapa.ufrgs.br/index.php?verb=pan&building=44' target='_blank'>SALA DE AULA 114 - PRÉDIO 43425 4</a></li><li>Quinta 15:30 - 17:10 (2)<br><a class='clicavel' href='http://mapa.ufrgs.br/index.php?verb=pan&building=44' target='_blank'>SALA DE AULA 114 - PRÉDIO 43425 4</a></li><li>Sexta 15:30 - 17:10 (2)<br><a class='clicavel' href='http://mapa.ufrgs.br/index.php?verb=pan&building=1885' target='_blank'>Laboratório de Ensino 102 - 43413 4</a></li><li><b></b></li></td>
                <td align="left"><li>Renan de Queiroz Maffei</li></td>
            </tr>
        </table>
    </fieldset>




<!-- Aqui encerra o conteudo desta página -->
  
</body>
</html>
