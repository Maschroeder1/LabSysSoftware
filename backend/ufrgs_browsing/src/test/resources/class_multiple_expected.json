{
  "timeslots": [
    {
      "classIdentifier": "A",
      "availableSlots": 1,
      "professors": [
        "MARA ABEL"
      ],
      "scheduledTimes": [
        {
          "day": "Segunda",
          "startTime": "13:30",
          "endTime": "15:10",
          "location": "SALA DE AULA 114 - PR�DIO 43425 4",
          "locationMap": "http://mapa.ufrgs.br/index.php?verb\u003dpan\u0026building\u003d44"
        },
        {
          "day": "Quarta",
          "startTime": "13:30",
          "endTime": "15:10",
          "location": "SALA DE AULA 114 - PR�DIO 43425 4",
          "locationMap": "http://mapa.ufrgs.br/index.php?verb\u003dpan\u0026building\u003d44"
        },
        {
          "day": "Quinta",
          "startTime": "13:30",
          "endTime": "15:10",
          "location": ""
        }
      ]
    },
    {
      "classIdentifier": "D",
      "availableSlots": 4,
      "professors": [
        "LUCAS MELLO SCHNORR",
        "MARCELO WALTER"
      ],
      "scheduledTimes": [
        {
          "day": "Segunda",
          "startTime": "15:30",
          "endTime": "17:10",
          "location": "SALA DE AULA 114 - PR�DIO 43425 4",
          "locationMap": "http://mapa.ufrgs.br/index.php?verb\u003dpan\u0026building\u003d44"
        },
        {
          "day": "Quarta",
          "startTime": "15:30",
          "endTime": "17:10",
          "location": "SALA DE AULA 114 - PR�DIO 43425 4",
          "locationMap": "http://mapa.ufrgs.br/index.php?verb\u003dpan\u0026building\u003d44"
        },
        {
          "day": "Quinta",
          "startTime": "15:30",
          "endTime": "17:10",
          "location": "Laborat�rio de Ensino 102 - 43413 4",
          "locationMap": "http://mapa.ufrgs.br/index.php?verb\u003dpan\u0026building\u003d1885"
        }
      ]
    },
    {
      "classIdentifier": "F",
      "availableSlots": 11,
      "professors": [
        "Joel Luis Carbonera",
        "THIAGO LOPES TRUGILLO DA SILVEIRA"
      ],
      "scheduledTimes": [
        {
          "day": "Ter�a",
          "startTime": "13:30",
          "endTime": "15:10",
          "location": "SALA DE AULA 114 - PR�DIO 43425 4",
          "locationMap": "http://mapa.ufrgs.br/index.php?verb\u003dpan\u0026building\u003d44"
        },
        {
          "day": "Quinta",
          "startTime": "13:30",
          "endTime": "15:10",
          "location": "SALA DE AULA 114 - PR�DIO 43425 4",
          "locationMap": "http://mapa.ufrgs.br/index.php?verb\u003dpan\u0026building\u003d44"
        },
        {
          "day": "Sexta",
          "startTime": "15:30",
          "endTime": "17:10",
          "location": "Laborat�rio de Ensino 101 - 43413 4",
          "locationMap": "http://mapa.ufrgs.br/index.php?verb\u003dpan\u0026building\u003d1885"
        }
      ]
    },
    {
      "classIdentifier": "H",
      "availableSlots": 10,
      "professors": [
        "Renan de Queiroz Maffei"
      ],
      "scheduledTimes": [
        {
          "day": "Ter�a",
          "startTime": "15:30",
          "endTime": "17:10",
          "location": "SALA DE AULA 114 - PR�DIO 43425 4",
          "locationMap": "http://mapa.ufrgs.br/index.php?verb\u003dpan\u0026building\u003d44"
        },
        {
          "day": "Quinta",
          "startTime": "15:30",
          "endTime": "17:10",
          "location": "SALA DE AULA 114 - PR�DIO 43425 4",
          "locationMap": "http://mapa.ufrgs.br/index.php?verb\u003dpan\u0026building\u003d44"
        },
        {
          "day": "Sexta",
          "startTime": "15:30",
          "endTime": "17:10",
          "location": "Laborat�rio de Ensino 102 - 43413 4",
          "locationMap": "http://mapa.ufrgs.br/index.php?verb\u003dpan\u0026building\u003d1885"
        }
      ]
    },
    {
      "classIdentifier": "B",
      "availableSlots": 0,
      "professors": [
        "MARA ABEL"
      ],
      "scheduledTimes": [
        {
          "day": "Segunda",
          "startTime": "13:30",
          "endTime": "15:10",
          "location": "SALA DE AULA 114 - PR�DIO 43425 4",
          "locationMap": "http://mapa.ufrgs.br/index.php?verb\u003dpan\u0026building\u003d44"
        },
        {
          "day": "Quarta",
          "startTime": "13:30",
          "endTime": "15:10",
          "location": "SALA DE AULA 114 - PR�DIO 43425 4",
          "locationMap": "http://mapa.ufrgs.br/index.php?verb\u003dpan\u0026building\u003d44"
        },
        {
          "day": "Quinta",
          "startTime": "15:30",
          "endTime": "17:10",
          "location": "Laborat�rio de Ensino 104 - 43413 4",
          "locationMap": "http://mapa.ufrgs.br/index.php?verb\u003dpan\u0026building\u003d1885"
        }
      ]
    },
    {
      "classIdentifier": "E",
      "availableSlots": 0,
      "professors": [
        "Joel Luis Carbonera",
        "THIAGO LOPES TRUGILLO DA SILVEIRA"
      ],
      "scheduledTimes": [
        {
          "day": "Ter�a",
          "startTime": "13:30",
          "endTime": "15:10",
          "location": "SALA DE AULA 114 - PR�DIO 43425 4",
          "locationMap": "http://mapa.ufrgs.br/index.php?verb\u003dpan\u0026building\u003d44"
        },
        {
          "day": "Quinta",
          "startTime": "13:30",
          "endTime": "15:10",
          "location": "SALA DE AULA 114 - PR�DIO 43425 4",
          "locationMap": "http://mapa.ufrgs.br/index.php?verb\u003dpan\u0026building\u003d44"
        },
        {
          "day": "Sexta",
          "startTime": "13:30",
          "endTime": "15:10",
          "location": "Laborat�rio de Ensino 101 - 43413 4",
          "locationMap": "http://mapa.ufrgs.br/index.php?verb\u003dpan\u0026building\u003d1885"
        }
      ]
    },
    {
      "classIdentifier": "G",
      "availableSlots": 14,
      "professors": [
        "Renan de Queiroz Maffei"
      ],
      "scheduledTimes": [
        {
          "day": "Ter�a",
          "startTime": "15:30",
          "endTime": "17:10",
          "location": "SALA DE AULA 114 - PR�DIO 43425 4",
          "locationMap": "http://mapa.ufrgs.br/index.php?verb\u003dpan\u0026building\u003d44"
        },
        {
          "day": "Quinta",
          "startTime": "15:30",
          "endTime": "17:10",
          "location": "SALA DE AULA 114 - PR�DIO 43425 4",
          "locationMap": "http://mapa.ufrgs.br/index.php?verb\u003dpan\u0026building\u003d44"
        },
        {
          "day": "Sexta",
          "startTime": "13:30",
          "endTime": "15:10",
          "location": "Laborat�rio de Ensino 102 - 43413 4",
          "locationMap": "http://mapa.ufrgs.br/index.php?verb\u003dpan\u0026building\u003d1885"
        }
      ]
    }
  ],
  "credits": 6
}